packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -413,7 +413,11 @@ export default {
         },
 
         shouldShowCheckoutForm () {
-            return !this.isLoading && !this.errorFormType;
+            return !this.isLoading && !this.errorFormType && !this.shouldShowAgeVerificationForm;
+        },
+
+        shouldShowAgeVerificationForm () {
+            return this.errors.some(error => error.code === 'DATE_OF_BIRTH_REQUIRED');
```

Comment:
can we get this from checkout issues?

Reply:
Checkout issues doesn't hold the code of the error. The code is the key. But I have just realised this does need changing because as basket is another domain, this will never become false. I will have to go with the original idea of having it in data.

Reply:
Actually we could remove the error when the user enters a correct DOB? 🤷‍♂️